new-frontiers-14/frontier-station-14
# Mix up atmos on planets

## About the PR
Added more gas mixes to expo, made it more simple to tell what they are unless you select unknown.
Selecting unknown will give you one few crazy mixes.

## Why / Balance
Add more options and make them more easy to select
Make the "fun" ones random selection

## How to test
Open on release or debug
Get expo ship
Look at expo selection
Go on one and see the gas on planet. 

## Media
![image](https://github.com/user-attachments/assets/38f01233-d851-44e3-98e0-e27182e78e2e)

## Requirements



## Breaking changes
N/A

**Changelog**
:cl:
- tweak: Changed and added more atmos selections on planets expeditions.
